### PR TITLE
Fix resource_pack_send has an additional arg that was forgotten

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: data/pc/1.17/protocol.json [hunk 1]
```diff
@@ -3085,6 +3085,13 @@
             {
               "name": "forced",
               "type": "bool"
+            },
+            {
+              "name": "promptMesage",
```
Comment (rom1504) [inline data/pc/1.17/protocol.json:3090]: typo

Comment (u9g) [reply data/pc/1.17/protocol.json]: good catch, fixed

**Review by halotroop2288 — APPROVED** [review]

File: data/pc/1.17/protocol.json [hunk 2]
```diff
@@ -3085,6 +3085,13 @@
             {
               "name": "forced",
               "type": "bool"
+            },
+            {
+              "name": "promptMessage",
+              "type": [
```
Comment (rom1504) [inline data/pc/1.17/protocol.json:3091]: that's wrong, option already contains the bool

Comment (u9g) [reply data/pc/1.17/protocol.json:3091]: I don't understand what you mean, I was trying to say this is an optional string arg, is that not what that means?

Comment (rom1504) [reply data/pc/1.17/protocol.json:3091]: yes but you need to remove the bool field

Comment (rom1504) [reply data/pc/1.17/protocol.json:3091]: option means bool + something
so you need to remove the bool above or there is 2 bool

Comment (u9g) [reply data/pc/1.17/protocol.json:3091]: by bool field, do you mean this one? https://github.com/PrismarineJS/minecraft-data/blob/b3b1b88b4af060af1157478616ca24a3e5dc638b/data/pc/1.17/protocol.json#L3086-L3087 if so, that should still be there, if not, I'm still not understanding you

**Pull request merged**